sqlite3/lib/src/wasm/file_system.dart

```diff
@@ -35,9 +37,16 @@ abstract class FileSystem {
   /// otherwise.
   void deleteFile(String path);
 
-  /// List all files stored in this file system.
+  /// Lists all files stored in this file system.
+  @Deprecated('Use files() instead')
   List<String> listFiles();
 
+  /// Lists all files stored in this file system.
+  List<String> files();
```

Comment:
While we're at it, I think `List<String> get files` would be more typical for a Dart API :)